.github/jobs/configure-checks/all.bats

```diff
@@ -324,12 +397,14 @@ compile_assertions_finished () {
   assert_line " * default user........: domjudge-bats-user"
   assert_line " * runguard user.......: domjudge-run"
   assert_line " * runguard group......: domjudge-run"
-  assert_line " * webserver group.....: www-data"
+  assert_regex "^ \* webserver group.....: (www-data|apache|nginx)$"
```

Comment:
```suggestion
  assert_regex "^ \* webserver group\.\.\.\.\.: (www-data|apache|nginx)$"
```

I think? Otherwise it matches any character right

Reply:
Yes, I think I missed that one.